lib/slugger.ts

```diff
@@ -53,6 +55,8 @@ export function plugin (schema: Schema, options?: SluggerOptions<any>) {
 
 export function wrap<D extends Document> (model: Model<D>): Model<D> {
 
+  const index: string = model.prototype.schema.plugins[0].opts.index;
```

Comment:
There might be multiple / different plugins hooked to the schema. So plugins[0] will not work in any case.